_assets/js/laws-regs-parser.js

```diff
@@ -111,9 +115,22 @@ function unHighlightText(e, divType) {
     section.classList.remove('highlight');
 }
 
+function openAccordions(section) {
+    Array.from(section.getElementsByTagName('details')).forEach(accordion => {
```

Comment:
supernit: a shorter way to write this, as querySelectorAll has forEach built in:

```suggestion
    section.querySelectorAll('details').forEach(accordion => {
```